modules/programs/convert_img/test_basic.py

```
@@ -0,0 +1,225 @@
+        try:
+            convert_img.convert_image_to_image(
+                input_path, output_path, "jpeg", quality=85
+            )
+            
+            if output_path.exists():
+                with Image.open(output_path) as result:
+                    if result.format == "JPEG" and result.size == (50, 50):
+                        print("✓ Basic PNG to JPEG conversion works")
+                        return True
+                    else:
+                        print(f"✗ Conversion result incorrect: format={result.format}, size={result.size}")
```

Flake8 (7.2.0)

[error] line too long (107 > 79 characters)

(E501)